### PR TITLE
fix: remove new exceptions

### PR DESCRIPTION
**Pull request by wey-gu**
## What type of PR is this?
- [x] bug
- [ ] feature
- [ ] enhancement

### CODE REVIEW

File: nebula3/gclient/net/Connection.py [hunk 1]
```diff
@@ -274,15 +266,15 @@ def close(self):
             self._connection._iprot.trans.close()
         except Exception as e:
             logger.error(
-                'Close connection to {}:{} failed:{}'.format(self._ip, self._port, e)
+                "Close connection to {}:{} failed:{}".format(self._ip, self._port, e)
```
Comment (wey-gu) [inline nebula3/gclient/net/Connection.py:269]: please ignore formatter's change, sorry

File: nebula3/gclient/net/Session.py [hunk 2]
```diff
@@ -25,6 +25,7 @@ def __init__(
         auth_result: AuthResult,
         pool,
         retry_connect=True,
+        retry_execute=False,
```
Comment (wey-gu) [inline nebula3/gclient/net/Session.py:28]: Let's make the execution error retry opt-in, thus to not surprise anyone.

File: nebula3/gclient/net/Session.py [hunk 3]
```diff
@@ -35,7 +36,8 @@ def __init__(
         :param auth_result: The result of the authentication process.
         :param pool: The pool object where the session was created.
         :param retry_connect: A boolean indicating whether to retry the connection if it fails.
-        :param retry_times: The number of times to retry the connection.
+        :param retry_execute: A boolean indicating whether to retry the execution if got execution error(-1005), by default False.
+        :param retry_times: The number of times to retry the connection/execution.
```
[the remark this answers is not in the document]
Comment (wey-gu) [reply nebula3/gclient/net/Session.py]: agree :), the retry times is already a newly introduced and it's feasible to be as flag with 0 value, done

**Review by HarrisChu — DISMISSED** [review]

File: nebula3/gclient/net/SessionPool.py [hunk 4]
```diff
@@ -170,34 +170,37 @@ def execute_parameter(self, stmt, params):
...
+            # Check for session validity based on error code
+            if resp.error_code() in [
+                ErrorCode.E_SESSION_INVALID,
+                ErrorCode.E_SESSION_TIMEOUT,
             ]:
                 self._active_sessions.remove(session)
                 session = self._get_idle_session()
                 if session is None:
-                    raise RuntimeError('Get session failed')
+                    logger.warning(
```
Comment (Nicole00) [inline nebula3/gclient/net/SessionPool.py:187]: why just print  warn log when session is None?
self._add_session_to_idle(session) will put a None session.

**Review by Nicole00 — APPROVED** [review]

File: nebula3/gclient/net/SessionPool.py (hunk 4 above)
Comment (wey-gu) [reply nebula3/gclient/net/SessionPool.py:187]: Sorry, silly me, fixed.

**Pull request merged**